### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by kdyla-griddynamics**
to run tests use:
npm test -e (browser name)

they work fine on firefox and chrome
safari has weird way to locate elements and does not find them easily by 'id' unless I use 'getElementById()' function. I will work on that once I'll implement allure.

you cannot run account creation on multiple browsers at once, because one will report that the email address is already registered. works fine separately, though you need to update email address after successful account creation

### CODE REVIEW

File: tests/createAccountFormTest.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            .verify.value('@stateDropdown', '38', "State set")
+            .setPostcode(postcode)
+            .verify.value('@postcode', postcode, "Postcode set")
+            .setMobile(mobile)
+            .verify.value('@mobile', mobile, "Mobile set")
+            .submitRegister(browser);
+
+        browser
+            .assert.urlContains('index.php?controller=my-account', "My account page url is correct")
+            .assert.visible('body[id="my-account"]', "My account page is visible");
```
Comment (Savostytskyi) [inline tests/createAccountFormTest.js]: Could you take all element locators from page object and just use variable in assert

File: page_objects/authenticationPage.js [hunk 2]
```diff
@@ -1,17 +1,35 @@
 module.exports = {
     url: "http://automationpractice.com/index.php?controller=authentication&back=my-account",
```
Comment (Savostytskyi) [inline page_objects/authenticationPage.js]: Lets imagine that we have different environments. Could you please create kind of property file and get url from there

**Review by Savostytskyi — APPROVED** [review]

File: tests/createAccountFormTest.js (hunk 1 above)
Comment (kdyla-griddynamics) [reply tests/createAccountFormTest.js:78]: done

File: page_objects/authenticationPage.js (hunk 2 above)
Comment (kdyla-griddynamics) [reply page_objects/authenticationPage.js]: done

**Pull request merged**